exporter/coralogixexporter/go.mod

```diff
@@ -0,0 +1,38 @@
+module coralogixexporter
+
+go 1.17
+
+require (
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.40.0
```

Comment:
This was bumped to https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/42a06f91316730473b00cee4284acaddd88bd3ff/go.mod#L111 can you update here?

Reply:
Yes, I will update the changes